### ISSUE TITLE
Setting variable `JUPYTER_ENABLE_LAB` to `1` causes some notebook pods to crash

### ISSUE THREAD

**Issue by cfchase** [opendatahub-io/odh-manifests#303]
**Describe the bug**
Setting variable `JUPYTER_ENABLE_LAB` to `1` causes minimal and tensorflow notebook pods into CrashLoopBackoff.

s1i-minimal-notebook:v0.0.4:
quay.io/thoth-station/s2i-minimal-notebook@sha256:b4be6d7f23396b3a7bc48225553aee5e177bb17f6cc906f5e61e6fd7a1904c05

s2i-tensorflow-notebook:v0.0.1:
quay.io/odh-jupyterhub/nbviewer@sha256:21ce1db27e96305718a06359c86139680b8138966c614b4c94796a788eeac98c

```
+ jupyter labhub --NotebookApp.default_url=/lab --config=/opt/app-root/etc/jupyter_notebook_config.py '--ip="0.0.0.0"' --port=8080
Traceback (most recent call last):
  File "/opt/app-root/bin/jupyter", line 8, in <module>
    sys.exit(main())
  File "/opt/app-root/lib/python3.6/site-packages/jupyter_core/command.py", line 247, in main
    command = _jupyter_abspath(subcommand)
  File "/opt/app-root/lib/python3.6/site-packages/jupyter_core/command.py", line 134, in _jupyter_abspath
    'Jupyter command `{}` not found.'.format(jupyter_subcommand)
Exception: Jupyter command `jupyter-labhub` not found.
```

**Steps To Reproduce**
1. JuptyerHub spawner
2. Select Minimal Notebook
3. Set variable name to `JUPYTER_ENABLE_LAB`
4. Set variable value to `1`
5. Click Spawn button
6. Watch Error for spawner and view pod log for detail.

**Expected behavior**
Notebook server should start with Jupyter Lab enabled.

**Workaround (if any)**
No JuptyerLab workaround for affected images.

**Open Data Hub Version**
0.9.0

**OpenShift Version**
Version #: OpenShift 4.6.8 (OSD on AWS)

**Additional context**
None.

**Comment by cfchase** [comment]
@sophwats @harshad16 @vpavlin

**Comment by harshad16** [comment]
On tracing through the error it seems this issue occurs with jupyter_core version 4.6.3 or lower and fixed in 4.7.0.
This package was updated in the s2i-minimal-notebook, however is v0.0.5 and above.

Will send out a PR to odh-manifests to update the version of s2i-minimal-notebook.
```
>$ podman run -it -p 8888:8888 quay.io/thoth-station/s2i-minimal-notebook:v0.0.7 
(app-root) jupyter --version
jupyter core     : 4.7.0
jupyter-notebook : 6.2.0
qtconsole        : not installed
ipython          : 7.16.1
ipykernel        : 5.4.3
jupyter client   : 6.1.11
jupyter lab      : 2.2.4
nbconvert        : 6.0.7
ipywidgets       : not installed
nbformat         : 5.1.2
traitlets        : 4.3.3
```
```
> $ podman run -it -p 8888:8888 quay.io/thoth-station/s2i-minimal-notebook:v0.0.4 (app-root) jupyter --version
jupyter core     : 4.6.3
jupyter-notebook : 6.0.3
qtconsole        : not installed
ipython          : 7.14.0
ipykernel        : 5.2.1
jupyter client   : 6.1.3
jupyter lab      : 2.1.2
nbconvert        : 5.6.1
ipywidgets       : not installed
nbformat         : 5.0.6
traitlets        : 4.3.3
````
